mlir-sycl/lib/Dialect/IR/SYCLOpsTypes.cpp

```diff
@@ -120,6 +120,32 @@ void mlir::sycl::printMemoryTargetMode(AsmPrinter &Printer,
   Printer << memoryTargetModeAsString(MemTargetMode);
 }
 
+std::string
+mlir::sycl::accessAddressSpaceAsString(mlir::sycl::AccessAddrSpace AccAddress) {
+  return std::to_string((int)AccAddress);
+}
+
+mlir::LogicalResult mlir::sycl::parseAccessAddrSpace(
+    mlir::AsmParser &Parser,
+    mlir::FailureOr<mlir::sycl::AccessAddrSpace> &AccAddress) {
+
```

Comment:
Can we add an assert that the target is SPIRV? 
As the current implementation assume that.

Reply:
That will be good, but not sure how to access target information here?

Reply:
I don't think we can do it here, as you point out.

Reply:
Too bad, how about we add a note here, so that we remember?

Reply:
Done